### ISSUE TITLE
Cohort Browser module

### ISSUE THREAD

**Issue by gowthamrao** [OHDSI/OhdsiShinyModules#1]
@jreps i would like to explore if its possible to make the cohort browser of cohort diagnostics  a module

**Comment by gowthamrao** [comment]
Input (required): 
[Cohort table](https://github.com/OHDSI/CohortDiagnostics/blob/master/inst/settings/resultsDataModelSpecification.csv#L10-L16) 

Input (Optional - any of):
[Cohort counts](https://github.com/OHDSI/CohortDiagnostics/blob/master/inst/settings/resultsDataModelSpecification.csv#L17-L20) 
OMOP vocabulary tables (multiple drop down)
[Orphan concepts](https://github.com/OHDSI/CohortDiagnostics/blob/master/inst/settings/resultsDataModelSpecification.csv#L126-L131)
[Resolved concepts](https://github.com/OHDSI/CohortDiagnostics/blob/master/inst/settings/resultsDataModelSpecification.csv#L142-L145)

Output: 

![image](https://user-images.githubusercontent.com/13936600/123785262-565c6a80-d8a6-11eb-9cd4-6bce985f10ce.png)

**Comment by gowthamrao** [comment]
Support both real-time and pre-computed output

**Comment by jreps** [comment]
@nhall6 - does your cohort module address this?

**Comment by nhall6** [comment]
@jreps no, not in its entirety. right now the only place we have orphan information is within CD module. Cohorts only has basic information, generation time, and attrition
